index.ios.js

```diff
@@ -15,39 +9,8 @@ import {
 export default class cdart extends Component {
   render() {
     return (
-      <View style={styles.container}>
```

Comment:
You deleted this stuff, but the render method must return an element. Please revise.